components/analytics/true_up_review.scss

```diff
@@ -0,0 +1,79 @@
+.TrueUpReview {
+    &__card {
+        width: 100%;
+        height: '463px';
+        border: 1px solid rgba(var(--sys-center-channel-color-rgb), 0.08);
+        background-color: var(--sys-center-channel-bg);
+        border-radius: 4px;
+        box-shadow: 0 2px 3px rgba(0, 0, 0, 0.08);
+        color: var(--sys-center-channel-color);
+    }
+
+    &__cardHeader {
+        display: flex;
+        align-items: center;
+        padding: 28px 32px 24px 32px;
+        border-bottom: 1px solid rgba(var(--sys-center-channel-color-rgb), 0.08);
+    }
+
+    &__cardHeaderText-top {
+        font-size: 16px;
+        font-weight: 600;
+        line-height: 24px;
+    }
+
+    &__cardBody {
+        display: flex;
+        flex-direction: column;
+        padding: 20px;
+    }
+
+    &__cardBody > * {
+        margin-bottom: 10px;
+    }
+
+    &__cardBody > svg {
+        margin-left: 15px;
+    }
+
+    &__dueDate {
+        :first-child {
+            color: rgba(var(--sys-center-channel-color-rgb), 0.64);
+        }
+    }
+
+    &__warning {
+        color: var(--warning-text);
+        font-size: 24px;
+    }
+
+    &__submit {
+        width: 181px;
+        height: 40px;
+        border: none;
+        border-radius: 4px;
+        box-shadow: none;
+        font-family: 'Open Sans';
+        font-size: 14px;
+        font-style: normal;
+        font-weight: 600;
+        letter-spacing: 0.02em;
+        line-height: 10px;
+
+        &--error {
+            width: 181px;
```

Comment:
I originally did the option you suggested instead of the way I went, however it seemed to apply all the classnames, so the element had: `class="btn btn-primary TrueUpReview__submit TrueUpReview__submit--error"`.

Maybe I did something wrong though, so I'll revisit just in-case.

Reply:
I'll also play around with the styles I've added to see if some can be omitted for the error state.

Reply:
Yeah, unfortunately,  this is what I see with:

```javascript
className={classNames('btn btn-primary TrueUpReview__submit', {'TrueUpReview__submit--error': trueUpReviewError})}

```

<img width="712" alt="image" src="https://user-images.githubusercontent.com/116016004/209998072-c8645ba8-8a75-4f3a-b3dd-fdf753142901.png">

Reply:
From the examples in npm:

```
classNames('foo', { bar: true }); // => 'foo bar'
```

I think we need to have the condition on both sides.